jwst/straylight/straylight.py

```diff
@@ -20,19 +20,52 @@
 
```

Comment:
I can't add a suggestion to unmodified lines, but the commented block at the top of the file should be changed to a triple-quoted docstring.